src/repology/repology_cli.py

```diff
@@ -43,7 +43,7 @@ def _pkg_str(str_obj):
     raise ArgumentTypeError("Value must be a non-empty string")
 
 
-def getargs():
+def getargs(str_args=None):
     """Parse command line arguments"""
```

Comment:
Does it make more sense to call this `cli`, and optionally have it accept `args`?

Also, please update the docstring, it should mention the fact you can pass in args as arguments to this function, rather than process args.

Reply:
Updated docsting with https://github.com/tiiuae/sbomnix/commit/8c85bcc44c5e8d76af562c9d3f06552b4a8d0e9b.

> _Does it make more sense to call this cli, and optionally have it accept args?_

In the new version, both parsing sys.argv, as well as (optionally) the string `str_args` is supported.